### PR TITLE
Portfolio: Add form validations

### PR DESCRIPTION
**Pull request by mateo951**
- Added js validations for email input
- Added fix for hiding error message after correction
- Added linters checks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,13 +193,19 @@ <h3 class="aboutme-font font-family">Skills</h3>
...
         new projects,
         so if you'd like to chat please get in touch.</h2>
-      <form class="flex-container form-content flex-form" action="https://formspree.io/f/xnqlbpyj" method="post">
-        <input class="input-layout" type="text" id="name" placeholder="Full Name" name="client_name" maxlength="30"
-          required autocomplete="off">
-        <input class="input-layout" type="email" id="mail" placeholder="Email Address" name="client_email" required
-          autocomplete="off">
-        <textarea class="input-layout" id="msg" placeholder="Enter text here" name="user_message" maxlength="500"
-          required></textarea>
+      <form class="flex-container form-content flex-form" action="https://formspree.io/f/xnqlbpyj" method="post" id="clientInfo">
```
Comment (Shaher-11) [inline index.html]: the form can be submitted with all the fields empty, kindly fix that and make sure the email must be the lower case only and all the fields are required

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @mateo951,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comment below:point_down:

>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html]: Done! Thanks!

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mateo, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾👍.
Congratulations! 🎉🥂

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**